app/models/submission.rb

```diff
@@ -33,7 +33,6 @@ class Submission < ApplicationRecord
   belongs_to :challenge
   belongs_to :phase, counter_cache: true
   belongs_to :submitter, class_name: 'User'
-  belongs_to :manager, class_name: 'User'
```

Comment:
Rails `belongs_to` associations are required by default. We could add the `optional: true` config option to the association to make it optional but I decided to remove it because I can't find any reason we need to have the manager associated with the submission and this field should probably be removed eventually. I'm not going to do that right now.

Reply:
This is causing an issue on staging because in the real world submissions are created without the `manager_id` which causes those records to fail validations on other kinds of updates in Rails.